cpu/esp32/include/sdkconfig_esp32.h

```diff
@@ -99,9 +99,14 @@ extern "C" {
 #define CONFIG_BROWNOUT_DET                     CONFIG_ESP32_BROWNOUT_DET
 
 #define CONFIG_ESP32_DEEP_SLEEP_WAKEUP_DELAY    2000
-#define CONFIG_ESP32_TRACEMEM_RESERVE_DRAM      0
+#define CONFIG_ESP32_TRACEMEM_RESERVE_DRAM      0x0
 #define CONFIG_ESP32_ULP_COPROC_RESERVE_MEM     0
 
+/**
+ * ESP32 specific sleep configuration (DO NOT CHANGE)
+ */
+#define CONFIG_ESP_SLEEP_RTC_BUS_ISO_WORKAROUND 1
```

Comment:
To be honest, I don't knwo. It's just a setting that comes from ESP-IDF's Kconfig without any explaination.
```
config ESP_SLEEP_RTC_BUS_ISO_WORKAROUND
        bool
        default y if IDF_TARGET_ESP32(=n) || IDF_TARGET_ESP32S2(=n) || IDF_TARGET_ESP32S3(=n)
```

Reply:
Interesting - better add a note that this is from the SDK

Reply:
Hm, but this is actually the case for all `CONFIG_*` definitions in the `sdkconfig*.h` files. They all define configurations that are only required for compiling ESP-IDF sources and are set to the default values of ESP-IDF's Kconfig. Maybe it is better to move them to a directory `cpu/esp32/esp-idf/include/config` so that it becomes better clear what these files are for.